xbmc/cores/IPlayer.h

```diff
@@ -159,9 +159,6 @@ enum EINTERLACEMETHOD
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL=14,
   VS_INTERLACEMETHOD_VDPAU_TEMPORAL_SPATIAL_HALF=15,
   VS_INTERLACEMETHOD_DEINTERLACE_HALF=16,
-  VS_INTERLACEMETHOD_DXVA_BOB = 17,
```

Comment:
not sure if this is supposed to have gaps. did you check CGUIDialogVideoSettings::InitializeSettings what happens when adding something undefined to a spinner

```
    CSettingInt *settingInterlaceMethod = AddSpinner(groupVideo, SETTING_VIDEO_INTERLACEMETHOD, 16038, 0, static_cast<int>(videoSettings.m_InterlaceMethod), entries);
```